test/hotspot/jtreg/compiler/ccp/TestRemoveUnreachableCCP.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ *
```

Comment:
Looking at other files, I think this empty line should be removed.

Reply:
ok, will do that